checklists/apim_checklist.en.json

```diff
@@ -309,6 +314,7 @@
             "text": "Disable Public Network Access",
             "guid": "d698adbd-3288-44cb-b10a-9b572da395ae",
             "severity": "High",
+            "graph": "resources | where type == "microsoft.apimanagement/service" | extend compliant = (properties.virtualNetworkType == "Internal") | distinct id, compliant",
```

Comment:
Double quotes in the query string break the JSON syntax, please use single quotes